draft-ietf-rtcweb-jsep.xml

```diff
@@ -1827,23 +1880,26 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
               <t>For RtpTransceivers that are still present, any
               "a=simulcast" line MUST stay the same.</t>
 
-              <t>If any MediaStreamTracks have been removed, either through
-              the removeStream method or by removing them from an added
-              MediaStream, their m= sections MUST be marked as recvonly by
-              changing the value of the <xref target="RFC3264"></xref>
-              directional attribute to "a=recvonly". The "a=msid", "a=ssrc",
-              and "a=ssrc-group" lines MUST be removed from the associated m=
-              sections.</t>
-
-              <t>If any MediaStreamTracks have been added, and there exist
-              m= sections of the appropriate media type with no associated
-              MediaStreamTracks (i.e. as described in the preceding paragraph),
-              those m= sections MUST be recycled by adding the new
-              MediaStreamTrack to the m= section. This is done by adding the
-              necessary "a=msid", "a=ssrc", and "a=ssrc-group" lines to the
-              recycled m= section, and removing the "a=recvonly" attribute.
-              </t>
-            </list></t>
+              <t>The directional attribute MAY change, depending on
+              the active state of the RtpSender or RtpReceiver, and
+              MUST have the value specified for initial offers in
+              <xref target="sec.initial-offers"></xref>.</t>
+
+              <t>If any RtpTransceiver has been stopped, the port
+              MUST be set to zero and the "a=msid", "a=ssrc", and
+              "a=ssrc-group" lines MUST be removed.</t>
+
+              <t>If any RtpTransceiver has been added, and there
+              exists a m= section with a zero port in the current
+              local description or the current remote description,
+              that m= section MUST be recycled by generating a m=
+              section for the added RtpTransceiver as if the m=
+              section were being added to session description, except
+              that instead of adding it, the generated m= section
+              replaces the m= section with a zero port.
+          </t>
+            </list>
+      </t>
 
           <t>If the initial offer was applied using setLocalDescription, and
           an answer from the remote side has been applied using
```

Comment:
Reading it, I think it can just be deleted, since it's taken care of elsewhere.  In other words, there are no more exceptions like these.